mne/report.py

```diff
@@ -1414,6 +1458,32 @@ def _render_cov(self, cov_fname, info_fname):
                                          show=show)
         return html
 
+    def _render_whitened_evoked(self, evoked_fname, noise_cov):
+        """Show whitened evoked.
+        """
+        global_id = self._get_id()
+
+        evokeds = read_evokeds(evoked_fname, verbose=False)
+
+        html = []
+        for ev in evokeds:
+            noise_cov = regularize(noise_cov, ev.info)
```

Comment:
This is not a good idea. please remove the regularization. we should show things how they are. Our cov estimation now takes care of this.